earthstar/effectbox.py

```diff
@@ -69,5 +71,7 @@ def main(fps, transition, animation, effect_addr, frame_addr):
         frame = engine.next_frame()
         frame = fc.virtual_to_physical(frame)
         frame_socket.send(frame.tobytes())
-        time.sleep(tick)
+        sleep_time = tick - (datetime.now() - start).total_seconds()
+        if sleep_time > 0:
+		    time.sleep(sleep_time)
```

Comment:
I replace the `datetime.now` calls with calls to `time.time` since those give us seconds straight up and are a lot faster and more what we want here (i.e. just relative times for this machine). Committed in 6cb6b39.